std/process.d

```diff
@@ -2073,19 +2087,26 @@ auto executeShell(in char[] command,
                   const string[string] env = null,
                   Config config = Config.none,
                   size_t maxOutput = size_t.max,
-                  in char[] workDir = null)
+                  in char[] workDir = null,
+                  string shellPath = nativeShell)
     @trusted //TODO: @safe
 {
-    return executeImpl!pipeShell(command, env, config, maxOutput, workDir);
+    return executeImpl!pipeShell(command,
+                                 env,
+                                 config,
+                                 maxOutput,
+                                 workDir,
+                                 shellPath);
 }
 
 // Does the actual work for execute() and executeShell().
-private auto executeImpl(alias pipeFunc, Cmd)(
+private auto executeImpl(alias pipeFunc, Cmd, TList...)(
     Cmd commandLine,
     const string[string] env = null,
     Config config = Config.none,
     size_t maxOutput = size_t.max,
-    in char[] workDir = null)
+    in char[] workDir = null,
+    TList values = TList.init)
```

Comment:
The default argument is unfortunately necessary because:
1) This must be the very last argument in the function because it is an unknown quantity of optional arguments
2) There are arguments before it that have default arguments; anything after an argument with a default value must also have a default value.